pages/index.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <div></div>
```

Comment:
Can you write something (even silly) in there? that way we know the server is up

Reply:
That's a good call out. Just ran it as it stood, and wasn't sure what to expect, but just a white page was a little disorientating